corsx/middleware_test.go

```diff
@@ -32,7 +32,7 @@ func TestContextualizedMiddleware(t *testing.T) {
 		require.NoError(t, err)
 		req.Header.Set("Origin", origin)
 		req.Header.Set("Access-Control-Request-Method", "DELETE")
-		req.Header.Set("Access-Control-Request-Headers", "origin")
+		req.Header.Set("Access-Control-Request-Headers", "")
```

Comment:
Apparently it is not a thing that the browser sets the origin header here:
https://github.com/rs/cors/issues/151